python/sandbox-gcp.config

```diff
@@ -0,0 +1,14 @@
+[sdk]
+workflow_packages=single_step,multi_step_linear
+python_venv=flytekit_venv
+
+[auth]
+assumable_iam_role:arn:aws:iam::123:role/test-role
+
+[platform]
+url=127.0.0.1:30081
+insecure=True
+cloud_provider=gcp
+
+[gcp]
+gcs_prefix=gs://bhram-flyte
```

Comment:
oops - this is our test bucket, can this be something more generic 😅

Reply:
I added it for you :), i will remove it and make it more generic,. but then you need to build the image